mapslice_test.go

```diff
@@ -50,3 +50,33 @@ func TestUnmarshal(t *testing.T) {
 		t.Errorf("expected: %s\ngot: %s", e, r)
 	}
 }
+
+func ExampleMapSlice_MarshalJSON() {
+	ms := MapSlice{
+		MapItem{"abc", 123, 0},
+		MapItem{"def", 456, 0},
+		MapItem{"ghi", 789, 0},
+	}
+
+	b, err := json.Marshal(ms)
+	if err != nil {
```

Comment:
If there is an error; likely you would want to exit with `panic()`, `log.Fatal()` or `os.Exit(1)`